docs/sources/panels-visualizations/visualizations/flame-graph/index.md

```diff
@@ -0,0 +1,86 @@
+---
+aliases:
+keywords:
+  - grafana
+  - dashboard
+  - documentation
+  - panels
+  - flame graph
+title: Flame graph
+weight: 850
+---
+
+# Flame graph panel
+
+> **Note:** This panel is currently in beta & behind the `flameGraph` feature toggle.
+
+## Flame graph
+
+The flame graph takes advantage of the hierarchical nature of profiling data. It condenses data into a format that allows you to easily see which code paths are consuming the most system resources.
+
+These resources are measured through profiles which aggregate that information into a format which is then sent to the flame graph visualization. For example, allocated objects or space when measuring memory.
+
+{{< figure src="/static/img/docs/flame-graph-panel/flame-graph.png" class="docs-image--no-shadow" max-width= "900px" caption="Figure 1 - Flame graph" >}}
```

Comment:
Shouldn't we use the markdown syntax for images? https://github.com/grafana/grafana/blob/main/contribute/documentation/documentation-markdown-guide.md#images

Reply:
Yeah! It looks like we have lots of other places to update this too :D

Reply:
True I always see this elsewhere and going to copy it and then I remember that we should do it some other way.